api/config/security.yml

```diff
@@ -130,7 +130,7 @@ security:
 
     # https://opgtransform.atlassian.net/wiki/spaces/DEPDS/pages/191299647/User+roles
```

Comment:
Does the page in the wiki require updating?

Reply:
Very good point (and I'd never seen this page before 😂 ) I've updated it now